macros/responseStatus.njk

```diff
@@ -0,0 +1,68 @@
+{% from "timeRemaining.njk" import timeRemaining %}
+
+{% macro responseStatusForDefendant(claim, url, isAfter4pm) %}
+
+  {% if claim.respondedAt %}
+    {% if claim.response.type === ResponseType.OWE_ALL_PAID_ALL %}
```

Comment:
Sharing macro without type classes is a bit problematic because you give something to community but you expect them to implement remaining bit.

I would like to start sharing domain model between CMC frontend applications as well since 'claim is a claim'.